src/FTPC.jl

```diff
@@ -480,6 +462,41 @@ end
 
 
 ##############################
+# PERFORM
+##############################
+
+function ftp_perform(ctxt::ConnContext, verbose::Bool, verbose_file)
+    resp = Response()
+    p_resp = pointer_from_objref(resp)
+
+    @ce_curl curl_easy_setopt CURLOPT_HEADERFUNCTION c_header_command_cb
+    @ce_curl curl_easy_setopt CURLOPT_HEADERDATA p_resp
+
+    libc_file = nothing
+    if verbose
+        @ce_curl curl_easy_setopt CURLOPT_VERBOSE Int64(1)
+        if isa(verbose_file, IOStream)
+            libc_file = Libc.FILE(verbose_file)
+            @ce_curl curl_easy_setopt CURLOPT_STDERR libc_file.ptr
+        end
+    else
+        @ce_curl curl_easy_setopt CURLOPT_VERBOSE Int64(0)
+    end
+
+    try
+        @ce_curl curl_easy_perform
+        process_response(ctxt, resp)
+    finally
+        if isa(libc_file, Libc.FILE)
+            close(libc_file)
+            @ce_curl curl_easy_setopt CURLOPT_STDERR Libc.FILE(RawFD(2), "w").ptr
```

Comment:
Try `Libc.FILE(STDERR.handle)` here. Not entirely sure it'll work but it definitely reads better.